app.js

```diff
@@ -4,66 +4,66 @@ if (process.env.NEW_RELIC_HOME) {
 } else {
   newrelic = {
     getBrowserTimingHeader: function () {
-      return "<!-- New Relic RUM disabled -->";
+      return '<!-- New Relic RUM disabled -->';
     }
   };
 }
 
-var express = require("express"),
-  domain = require("domain"),
-  cluster = require("cluster"),
-  habitat = require("habitat"),
-  helmet = require("helmet"),
-  http = require("http"),
-  middleware = require("./lib/middleware"),
-  nunjucks = require("nunjucks"),
-  path = require("path"),
-  lessMiddleWare = require("less-middleware"),
-  i18n = require("webmaker-i18n"),
-  WebmakerAuth = require("webmaker-auth"),
-  navigation = require("./navigation"),
-  rtltrForLess = require("rtltr-for-less"),
-  markdown = require("markdown").markdown,
-  proxy = require("proxy-middleware"),
-  url = require("url");
+var express = require('express'),
+  domain = require('domain'),
+  cluster = require('cluster'),
+  habitat = require('habitat'),
+  helmet = require('helmet'),
+  http = require('http'),
+  middleware = require('./lib/middleware'),
+  nunjucks = require('nunjucks'),
+  path = require('path'),
+  lessMiddleWare = require('less-middleware'),
+  i18n = require('webmaker-i18n'),
+  WebmakerAuth = require('webmaker-auth'),
+  navigation = require('./navigation'),
+  rtltrForLess = require('rtltr-for-less'),
+  markdown = require('markdown').markdown,
+  proxy = require('proxy-middleware'),
+  url = require('url');
 
 habitat.load();
 
 var app = express(),
-  env = new habitat(),
+  env = habitat(),
```

Comment:
no, this should definitely be  a `new ..` constructor, but the constructor needs to be `Habitat` with a capital